### PR TITLE
Small updates

### PR DESCRIPTION
**Pull request by KostyaSha**
https://github.com/KostyaSha/yet-another-docker-plugin/pull/84
Fluent code and imports.

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/cloudstats/CloudStatistics.java [hunk 1]
```diff
@@ -264,7 +274,7 @@ private XmlFile getConfigFile() {
     @Extension
     public static class ProvisioningListener extends CloudProvisioningListener {
 
-        private final CloudStatistics stats = CloudStatistics.get();
+        private final CloudStatistics stats = getCloudStatistics();
```
Comment (olivergondza) [inline src/main/java/org/jenkinsci/plugins/cloudstats/CloudStatistics.java]: This seems like a personal preference. Adding static method imports just to save one or couple of characters is not what I prefer. I am ok to add these to the top level class for convenience but without deprecating the old methods.

Comment (KostyaSha) [inline src/main/java/org/jenkinsci/plugins/cloudstats/CloudStatistics.java]: That not a preference, that how human friendly and readable code should look like. I spent multiple hours reading this limited number of classes and feel impression like Perl.  Nested classes everywhere understandable only for author.  `CloudStatistics.get()` i think that at all doesn't follow getter/setters convention, getter doesn't answer what exactly i can get from this class. Static import allow to have clean and clear readable code. The same is about `getId` `Id` class. You may end like https://github.com/jenkinsci/ldap-plugin/blob/master/src/main/java/hudson/security/LDAPSecurityRealm.java where everything is in one class.

Comment (olivergondza) [inline src/main/java/org/jenkinsci/plugins/cloudstats/CloudStatistics.java]: > That not a preference, that how human friendly and readable code should look like.

Of course it is, you just refuse to admit other people do not share your religion :smile:.

Comment (KostyaSha) [inline src/main/java/org/jenkinsci/plugins/cloudstats/CloudStatistics.java]: There is no religion, things that i do do all modern frameworks and libraries.

**Pull request closed without merging**